src/components/pages/Auth.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import client from 'api/client';
+import { issueJWT } from 'api/auth';
+import useQuery from 'hooks';
+import { RouteComponentProps, withRouter } from 'react-router';
+
+const Auth = ({ history }: RouteComponentProps) => {
+  const tempToken = useQuery().get('authorizationToken');
+
+  useEffect(() => {
+    if (tempToken) {
+      issueJWT(tempToken)
```

Comment:
동일한 동작이지만... early return 해주는게 함수 뎁스 하나를 줄일 수 있을 것 같아요 ㅎㅎ
```suggestion
  useEffect(() => {
    if (tempToken === undefined) {
      return;
    }
    issueJWT(tempToken)
```